### PR TITLE
Readme, Logo and Badges Added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
A generic template for README.md will be added. This template contains a table of contents covering the necessary information for the readme page. Also, some badges are added right below the logo.

There are some commented parts that should be taken care of before the version 0.1 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,92 @@
...
-### TODOs:
-1- refactoring folders & support for different frameworks 
-<br/>
-2- udpate doc string 
-<br/>
-3- error handling 
-<br/>
-4- readme of version 1
+<div align="center">
+    <img src="https://github.com/openscilab/pymilo/raw/master/otherfiles/logo.png">
```
Comment (sepandhaghighi) [inline README.md:2]: add width="550"

File: README.md [hunk 2]
```diff
@@ -1,19 +1,92 @@
...
+</div>
+
+----------
+
+## Table of contents
+
+* [Overview](https://github.com/openscilab/pymilo#overview)
+* [Installation](https://github.com/openscilab/pymilo#installation)
+* [Usage](https://github.com/openscilab/pymilo#usage)
+  <!-- * [Scikit-learn](https://github.com/openscilab/pymilo#sciki-learn) -->
```
Comment (sepandhaghighi) [inline README.md:32]: Remove comments

File: README.md [hunk 3]
```diff
@@ -1,19 +1,92 @@
...
+
+## Usage
+
+
+## Issues & bug reports
+
+1. Fill an issue and describe it. We'll check it ASAP!
+    - Please complete the issue template
+2. Discord : [https://discord.gg/mtuMS8AjDS](https://discord.gg/mtuMS8AjDS)
+<!-- 3. Website : [https://www.ascii-art.site](https://www.ascii-art.site) -->
```
Comment (sepandhaghighi) [inline README.md:79]: Remove comments

File: README.md [hunk 4]
```diff
@@ -1,19 +1,92 @@
...
+<!-- * [Cite](https://github.com/openscilab/pymilo#cite) -->
+* [Authors](https://github.com/openscilab/pymilo/blob/master/AUTHORS.md)
+* [License](https://github.com/openscilab/pymilo/blob/master/LICENSE)
+* [Show Your Support](https://github.com/openscilab/pymilo#show-your-support)
+* [Changelog](https://github.com/openscilab/pymilo/blob/master/CHANGELOG.md)
+* [Code of Conduct](https://github.com/openscilab/pymilo/blob/master/.github/CODE_OF_CONDUCT.md)
+
+
+## Overview
+
```
Comment (sepandhaghighi) [inline README.md:46]: Add code quality and CI tables. We will fill them later

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
Please take a look at my comments.
Also, resolve the conflicts 💯

**Comment by AHReccese** [comment]
@sadrasabouri Great job Sadra.
I applied some tiny fixes:
1- One for centering the overview tables and 
2- the other one is to modify the name of the main branch referring to(main instead of master(which is the main in pycm :))).
Thanks.

**Review by AHReccese — APPROVED** [review]
LGTM

**Comment by sadrasabouri** [comment]
> @sadrasabouri Great job Sadra. I applied some tiny fixes: 1- One for centering the overview tables and 2- the other one is to modify the name of the main branch referring to(main instead of master(which is the main in pycm :))). Thanks.

Great, thanks.

**Comment by sepandhaghighi** [comment]
> @sadrasabouri Great job Sadra. I applied some tiny fixes: 1- One for centering the overview tables and 2- the other one is to modify the name of the main branch referring to(main instead of master(which is the main in pycm :))). Thanks.

@AHReccese 
Thanks for your efforts 💯 
I disagree with that centering personally. It would be better to align to the right because we will probably add some text there later.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@AHReccese take a look at my comments 📖

**Comment by AHReccese** [comment]
rollbacked to left-aligned according to Sepand's suggestion.

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯

**Pull request merged**